.github/workflows/ci.yml

```diff
@@ -90,20 +90,40 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Determine tag
+      - name: Determine Docker Tag
         uses: haya14busa/action-cond@v1
         id: imagetag
         with:
           cond: ${{ github.event_name == 'pull_request' }}
           if_true: ${{ github.sha }}
-          if_false: "master"
+          if_false: 'master'
 
-      - name: Build
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: dexidp/dex
-          tags: ${{ steps.imagetag.outputs.value }}
-          add_git_labels: true
-          push: ${{ github.event_name == 'push' }}
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+          version: latest
+          driver-opts: image=moby/buildkit:master
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and Push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          push: true
```

Comment:
We don't want to push images other than the master (please see the previous version).

Reply:
No problem, easy fix.